app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<section class="container">
+  <h2>Sign up</h2>
+
+  <div class="form">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="field">
+        <%= f.label :email, class: 'field-label' %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'field-input' %>
+      </div>
+      <div class="field">
+        <%= f.label :name, class: 'field-label' %><br />
+        <%= f.text_field :name, class: 'field-input' %>
+      </div>
+
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %>
+      <div class="field">
+        <%= f.label :password, class: 'field-label' %>
+        <%= f.password_field :password, autocomplete: "new-password", class: 'field-input' %>
+      </div>
+
+      <div class="field">
+        <%= f.label :password_confirmation, class: 'field-label' %><br />
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'field-input' %>
+      </div>
+
+      <div class="actions">
+        <%= f.submit "Sign up", class: 'signup-btn' %>
+      </div>
+    <% end %>
+
+    <div class="shared-links">
+      <%= render "devise/shared/links" %>
+    </div>
+  </div>
+
+</section>
```

Comment:
- [ ] When ı signup ı am getting an error in the terminal and the page does not render anything. After that If ı click sign in or public recipes on the navbar ı can log in. It will be good log in automatically if ı fill out the form and click on the signup
button. Kindly fix it.
![image](https://user-images.githubusercontent.com/38635158/181830303-d588bc0f-b388-466b-8a32-d24534a41e8d.png)